templates/asp.net-core-.net-framework.yml

```diff
@@ -6,29 +6,51 @@
 trigger:
 - master
 
-pool:
-  vmImage: 'windows-latest'
+stages:
```

Comment:
There are so many options to deploy a .net core app. It looks like the intent of this template is to provide enough of a boiler plate so that one can later add the script/task more easily through the editor. I agree that this is a good step.